### PR TITLE
MS Personal Standard deduction

### LINKED ISSUE #2318: MS Individual Standard Deduction
$4,600 married (filing jointly or combined return, spouse died)
$4,600 married (filing separately, each get $$2,300) 
$3,400 Head of Family
$2,300 Single

reference: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=7a1c7bb2-7110-478b-aa87-7a0f540809eb&config=00JABhZDIzMTViZS04NjcxLTQ1MDItOTllOS03MDg0ZTQxYzU4ZTQKAFBvZENhdGFsb2f8inKxYiqNVSihJeNKRlUp&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A67NM-4K73-CGX8-048B-00008-00&pdcontentcomponentid=234190&pdteaserkey=sr0&pditab=allpods&ecomp=8s65kkk&earg=sr0&prid=e8257a64-6769-47fa-91b1-982d7b418151 3(b)(i - iv)

### PR DESCRIPTION
**Pull request by Xinyi-Shao**
Fixes #2318

create MS personal standard deduction

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ms/tax/income/deductions/standard/amount.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  label: Mississippi personal standard deduction
+  reference: 
+    - title: Mississippi State FORMS 80-105 AND 80-205
+      href: https://www.dor.ms.gov/sites/default/files/Forms/Individual/80100221.pdf#page=5
+    - title: § 27-7-17. Individual nonbusiness deductions. 3(i)(i - iv)
+      href: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=7a1c7bb2-7110-478b-aa87-7a0f540809eb&config=00JABhZDIzMTViZS04NjcxLTQ1MDItOTllOS03MDg0ZTQxYzU4ZTQKAFBvZENhdGFsb2f8inKxYiqNVSihJeNKRlUp&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A67NM-4K73-CGX8-048B-00008-00&pdcontentcomponentid=234190&pdteaserkey=sr0&pditab=allpods&ecomp=8s65kkk&earg=sr0&prid=e8257a64-6769-47fa-91b1-982d7b418151
+  breakdown:
+    - filing_status
+SINGLE:
+  2020-01-01: 2_300
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ms/tax/income/deductions/standard/amount.yaml:13]:
```suggestion
  1999-01-01: 2_300
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ms/tax/income/deductions/standard/amount.yaml:13]: same for other values in yaml file

File: policyengine_us/variables/gov/states/ms/tax/income/deductions/standard/ms_personal_standard_deduction.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from policyengine_us.model_api import *
+
+
+class ms_personal_standard_deduction(Variable):
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ms/tax/income/deductions/standard/ms_personal_standard_deduction.py:4]:
```suggestion
class ms_standard_deduction(Variable):
```

Comment (PavelMakarchuk) [reply policyengine_us/variables/gov/states/ms/tax/income/deductions/standard/ms_personal_standard_deduction.py:4]: change variable and test file names to `ms_standard_deduction`

File: policyengine_us/variables/gov/states/ms/tax/income/deductions/standard/ms_personal_standard_deduction.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    value_type = float
+    entity = TaxUnit
+    label = "Mississippi personal standard deduction"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ms/tax/income/deductions/standard/ms_personal_standard_deduction.py:7]:
```suggestion
    label = "Mississippi standard deduction"
```

File: changelog_entry.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+    - Mississippi personal standard deduction.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - Mississippi standard deduction.
```

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**